lib/trie/trie_test.go

```diff
@@ -3548,3 +3548,40 @@ func Benchmark_concatSlices(b *testing.B) {
 		}
 	})
 }
+
+//ParityTech/trie proofs are incompatible with gossamer/lib/trie #2329
```

Comment:
nit add a space, otherwise it's meant to be a machine-readable comment thingy (like `//go:generate` etc.)

```suggestion
// ParityTech/trie proofs are incompatible with gossamer/lib/trie #2329
```

Reply:
Ohh, good point, will fix it.

Forgive my ignorance, but what is `nit` ?

Reply:
> Reviewers should always feel free to leave comments expressing that something could be better, but if it’s not very important, prefix it with something like “Nit: “ to let the author know that it’s just a point of polish that they could choose to ignore.

From [Google's Standard of Code Review](https://google.github.io/eng-practices/review/reviewer/standard.html)

Also from [Stackoverflow](https://stackoverflow.com/a/27810540](https://stackoverflow.com/a/27810540)

> In the context of "nit: removed whitespace," I would assume that the term is short for nit-pick. That is, a small change that may not be very important, but is technically correct.

And some [Wikipedia Nitpicking definition](https://en.wikipedia.org/wiki/Nitpicking)

I have no idea how I learnt this word though, but I pretty much type it at least once a day :smile:

Reply:
nice one, thanks for letting me know :)